iso.c

```diff
@@ -209,9 +210,7 @@ iso_connect(char *server, char *username, char *domain, char *password,
 	uint8 code;
 	uint32 neg_proto;
 
-	g_negotiate_rdp_protocol = True;
-
-	neg_proto = PROTOCOL_SSL;
+	neg_proto = *selected_protocol;
```

Comment:
I think that you should not change this line in this PR.

Reply:
done